### PR TITLE
Monitor govc session to ensure full VMDK downloads

### PR DESCRIPTION
**Pull request by abhay-krishna**
We were facing issues with importing the Ubuntu 22.04 OVAs built in CI, with the most common error being
```console
Reason: Error transferring file ubuntu-2204-kube-v1.28.4-1.vmdk to ds:///vmfs/volumes/vsan:.... Reason: Unable to parse disk image while uploading to: ds:///vmfs/volumes/vsan:...
```
This pointed to an issue with the VMDK itself. To build the OVA, we download the OVF, VMDK and NVRAM from the vCenter Content Library in the Packer post-processor step and then pass them as inputs to a script which packages them into an OVA. Since December 15 or so, we noticed that the OVAs being uploaded to S3 had shrunk in size compared to previous uploads (~6 GB before vs ~2 GB after). Upon downloading the OVA, extracting the VMDK and inspecting it, we encountered the following error.
```console
$ qemu-img check ubuntu-2204-kube-v1.28.4-1.vmdk       
qemu-img: Could not open 'ubuntu-2204-kube-v1.28.4-1.vmdk': Invalid footer
```
This indicates that the VMDK is corrupt and from the size reduction, we hypothesized that it's possibly not being downloaded fully from the Content Library during the image build. To investigate if this was caused due to a vCenter session timeout issue, we monitored the sessions using
```console
$ while true; do govc library.session.ls; done
```
Surprisingly, keeping this session monitoring alive during the VMDK download caused it to download the entire 6GB VMDK instead of cutting short at ~2 GB, and the downloaded VMDK had no errors when inspected.
```console
$ qemu-img check ubuntu-2204-kube-v1.28.4-1.vmdk
No errors were found on the image.
```
The import of this VMDK also worked fine. So we are adding this workaround to the image build by running the session monitor as a background process until the OVA is built.

I've also bumped EKS-D releases and made some fixes to the EKS-A admin AMI while I'm at it. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by jaxesn — COMMENTED** [review]
I hate it but I love it at the same time!

Thanks for compiling the background in the description, this will help us out in a few months when we are reminded of this hack and cant remember what we were thinking!

File: projects/aws/eks-a-admin-image/build/export-ami-to-s3.sh [hunk 1]
```diff
@@ -122,4 +122,4 @@ done
 INSPECTOR_FINDINGS_REPORT_ID=$(aws inspector2 create-findings-report --filter-criteria '{"resourceId": [{"comparison": "EQUALS", "value": "'"$INSTANCE_ID"'"}], "findingStatus": [{"comparison": "EQUALS", "value": "ACTIVE"}]}' --report-format CSV --s3-destination '{"bucketName": "'"$SNOW_ADMIN_IMAGE_INSPECTOR_BUCKET"'", "kmsKeyArn": "'"$SNOW_ADMIN_IMAGE_INSPECTOR_KMS_KEY_ARN"'"}' --query "reportId" --output text) 
 
 # Make findings report public 
-aws s3api put-object-acl --bucket $SNOW_ADMIN_IMAGE_INSPECTOR_BUCKET --key $INSPECTOR_FINDINGS_REPORT_ID --acl public-read
+aws s3api put-object-acl --bucket $SNOW_ADMIN_IMAGE_INSPECTOR_BUCKET --key $INSPECTOR_FINDINGS_REPORT_ID.csv --acl public-read
```
Comment (abhay-krishna) [inline projects/aws/eks-a-admin-image/build/export-ami-to-s3.sh:125]: Fixes this error
```console
An error occurred (NoSuchKey) when calling the PutObjectAcl operation: The specified key does not exist.
```

File: projects/aws/eks-a-admin-image/build/validate-release-manifest-version.sh [hunk 2]
```diff
@@ -12,8 +12,8 @@ n=0
 max_attempts=120
 delay=30
 while true; do
-    LATEST_EKSA_VERSION_S3=$(curl $EKSA_RELEASE_MANIFEST_S3_URL | yq e '.spec.latestVersion' -)
-    LATEST_EKSA_VERSION_CLOUDFRONT=$(curl $EKSA_RELEASE_MANIFEST_CDN_URL | yq e '.spec.latestVersion' -)
+    LATEST_EKSA_VERSION_S3=$(curl $EKSA_RELEASE_MANIFEST_S3_URL | yq e '.spec.releases[-1].version' -)
```
Comment (abhay-krishna) [inline projects/aws/eks-a-admin-image/build/validate-release-manifest-version.sh:15]: This change is needed after we changed the logic to set the latest version in the release process. It potentially fixes the error
```console
Error: invalid version v0.17.5, no matching release found
```

File: EKSD_LATEST_RELEASES [hunk 3]
```diff
@@ -1,38 +1,16 @@
-releases:
```
Comment (abhay-krishna) [inline EKSD_LATEST_RELEASES:1]: Decided to clean this up as I didn't see any reason to maintain the deprecated versions.

Comment (jaxesn) [reply EKSD_LATEST_RELEASES:1]: i think we may a couple of them, for the previous pause image tag logic. id revert this honestly and do this separately if at all

**Review by jaxesn — APPROVED** [review]

**Pull request merged**